package-lock.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "anchor-js",
-  "version": "4.2.2",
+  "version": "4.3.0",
```

Comment:
BTW when you bump the version, this needs to be updated too. I personally get everything ready and then use `npm version` which handles the version bump + the git tag creating. Then you only need to push both.